Detectors/TRD/workflow/src/TRDDigitWriterSpec.cxx

```diff
@@ -0,0 +1,46 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef STEER_DIGITIZERWORKFLOW_SRC_TRDDIGITWRITERSPEC_H_
+#define STEER_DIGITIZERWORKFLOW_SRC_TRDDIGITWRITERSPEC_H_
+
+#include "Framework/DataProcessorSpec.h"
+#include "DPLUtils/MakeRootTreeWriterSpec.h"
+#include "Framework/InputSpec.h"
+#include "TRDBase/Digit.h"
+#include "DataFormatsTRD/TriggerRecord.h"
+#include <SimulationDataFormat/MCTruthContainer.h>
+#include "TRDBase/MCLabel.h"
+
```

Comment:
I am a bit confused by this PR. This seems like new source code implementing some function but I do not see the corresponding header (in the include list).

Reply:
It is merely the implementation of a method (getTRDDigitWriterSpec) defined in the header file. The header file would have to be included elsewhere if the method is to be called, right?

In anycase this PR is not really going to really solve anything as the trapsim PR and other classes in DigitizerWorkflow descend from o2::framework::task and one cant forward declare base classes.

Reply:
I think you need to include the header for the symbols you are implementing for consistency.